client/components/tinymce/plugins/wpcom-paste-track/plugin.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import tinymce from 'tinymce/tinymce';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+const debug = debugFactory( 'calypso:tinymce-plugins:wpcom-paste-track' );
+
+const SOURCE_GOOGLE_DOCS = 'google_docs';
+const SOURCE_UNKNOWN = 'unknown';
+
+function pasteTrack( editor ) {
```

Comment:
the naming here is backwards. it reads as a noun. probably it should be a verb since it is there _to do something_.

`trackPaste` for example